src/components/NodeLink/NodeLink.vue

```diff
@@ -69,6 +73,18 @@ export default {
         edgeWidthKey: undefined,
       })
     },
+    renderNested: {
+      type: Boolean,
+      default: false
+    },
+    nestedBarVariables: {
+      type: Array[String],
```

Comment:
I don't think this is valid JavaScript. To accomplish the intent you would change this to `type: Array,`, and then add a `validator` section that looks something like

```javascript
validator: (prop) => prop.every((item) => typeof item === 'string'),
```

The other option here is to use TypeScript, which makes it easier to do this sort of check without the runtime validator.

Reply:
I was trying to be more specific with types. Are you in favor of just switching all these to Array instead of Array[String] since it will reduce complexity?

Reply:
It's good to be specific with types, but what I'm saying is, `Array[String]` is not a valid JavaScript type. Setting it to `Array` *is* valid, though not very specific; adding that `validator` to the prop definition will let you validate that every element of the prop is in fact a string.

This may explain why Vue is telling you that your default value needs to be a function (though that's just a guess). Technically, `Array[String]` evaluates to `undefined`, so maybe that's somehow making Vue infer that the prop is a function type. (I wouldn't put money on this explanation.)

The better approach would be to set the type as `Array as string[]` but that would require TypeScript which of course is a major upgrade to make. So at least for now I'd try out the `Array`+`validator` approach.

Reply:
Should be fixed in the recent commits